RecoLocalCalo/HcalRecAlgos/python/hcalRecAlgoESProd_cfi.py

```diff
@@ -85,4 +86,32 @@
         ),
     RecoveredRecHitBits = cms.vstring('TimingAddedBit','TimingSubtractedBit'),
     DropChannelStatusBits = cms.vstring('HcalCellMask','HcalCellOff', 'HcalCellDead')
+) 
+
+# This is to use phase1 flags defined in DataFormats/METReco/interface/HcalPhase1FlagLabels.h
+from Configuration.Eras.Modifier_run2_HCAL_2017_cff import run2_HCAL_2017
+run2_HCAL_2017.toModify(hcalRecAlgos,
+    phase = cms.uint32(1),
+    SeverityLevels = {
+        0 : dict( RecHitFlags = cms.vstring('TimingFromTDC', 
```

Comment:
@abdoulline 

You are right. After removing the explicit assignments for SevLvl=0, I can confirm that the discrepant distributions become identical. I will commit the change shortly.